pkg/webreg/webreg.go

```diff
@@ -1156,6 +1156,17 @@ The configuration can also override a workflow field with a <a href="#step">full
 
 {{ yamlSyntax (index . "configExample3") }}
 
+<h3 id="allow-skip-on-success"><a href="#allow-skip-on-success">Optional skipping of post steps</a></h3>
```

Comment:
Nit: can we title-case this, make it a subsection and make the phrase more declarative?

```html
 <h2 id="allow-skip-on-success"><a href="#allow-skip-on-success">Options to Change Control Flow</a></h2>
```

Reply:
Do you want it exactly as you suggested? The "allow-skip-on-success" doesn't really match the title anymore which seems to be more general in your suggestion

Reply:
Anyway. Pushed. Thanks for reviews!